src/main/java/com/sourcegraph/common/AuthorizationUtil.java

```diff
@@ -4,11 +4,7 @@
 
 public class AuthorizationUtil {
   public static boolean isValidAccessToken(@NotNull String accessToken) {
-    return accessToken.isEmpty()
-        || accessToken.length() == 40
-        || (accessToken.startsWith("sgp_") && accessToken.length() == 44)
-        // See https://docs.google.com/document/d/1aC4gHB8Q5lurwVhc8SCxznR0blNJMKo7yIkpP7WShno
-        || (accessToken.startsWith("sgp_") && accessToken.length() == 61)
-        || (accessToken.startsWith("sgph_") && accessToken.length() == 62);
+    // Sourcegraph access token formats: https://docs.sourcegraph.com/dev/security/secret_formats
```

Comment:
<img width="702" alt="image" src="https://github.com/sourcegraph/jetbrains/assets/19799111/4aaa406c-d78f-4e50-b495-db83c8d99b99">

Should the page be yet available? 🤔

Reply:
Not quite yet I'm afraid!

> URL in comment is pending https://github.com/sourcegraph/sourcegraph/pull/57722 merging.

Reply:
PR has merged and link now works @mkondratek https://docs.sourcegraph.com/dev/security/secret_formats

Reply:
Thanks! It looks good. 

I am curious... what about `sgd_`, `slk_`, `sgs_` tokens? They are mentioned in the docs. However, they are invalid in the sense of this method. Is it ok? 🤔

Reply:
Yep not a problem. The jetbrains extension only needs to use access tokens (`sgp_`) which are used to talk to a sourcegraph instance. These other tokens are used for various other backend authentication purposes so don't need to be supported here 😄